src/app/pages/jobs/components/job-logs-sidebar/job-logs-sidebar.component.ts

```diff
@@ -9,10 +9,21 @@ import { Job } from 'app/interfaces/job.interface';
   styleUrls: ['./job-logs-sidebar.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class JobLogsSidebarComponent {
+export class JobLogsSidebarComponent implements OnChanges {
   @Input() job: Job;
+  @Input() viewType: string;
   @Output() closed = new EventEmitter<void>();
 
+  constructor(private cdr: ChangeDetectorRef) {}
+
+  ngOnChanges(): void {
+    const argument = document.getElementById('argument') as HTMLInputElement;
```

Comment:
Why not use JSON pipe inside template like `{{ jobs.arguments | json }}` instead of modifying DOM from the component?